src/txn/txn_ckpt.c

```diff
@@ -1397,30 +1341,26 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
      * Ensure that the metadata changes are durable before the checkpoint is resolved. Do this by
      * either checkpointing the metadata or syncing the log file. Recovery relies on the checkpoint
      * LSN in the metadata only being updated by full checkpoints so only checkpoint the metadata
-     * for full or non-logged checkpoints.
+     * for full or non-logged checkpoints. Note, all checkpoints are full now.
```

Comment:
I think the new note is confusing now that the notion of non-full checkpoints has been removed from the codebase. What's the benefit of considering full vs. targeted checkpoints here? Perhaps there's another way to get that across.

Reply:
Done. I have rewritten it to be clear but not lose information.